submissions/israelOladosu/database/transportationQuery.sql

```diff
@@ -0,0 +1,44 @@
+-- Total number of passengers who suvrvived
+SELECT COUNT(survived) FROM accidents WHERE survived = 1;
```

Comment:
Write the result of what you got using a comment. Note your answer should be a number